lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -34,6 +34,10 @@ def self.cli_to_symbol(cli)
         :aes
       when '7', 7
         :cisco_type_7
+      when 'clear'
+        :clear
```

Comment:
Would be nice to use the existing `:cleartext` value here if possible - having `:clear` and `:cleartext` mean different things seems confusing. Similarly, do we know which encryption 'type' corresponds to `:encrypted`?